components/about/OurInvestors.vue

```diff
@@ -101,7 +101,7 @@ export default {
           image: 'images/investors/paradigm-shift.png',
           name: 'Paradigm Shift VC',
           description:
-            'Paradigm Shift Capital is a venture capital and private equity firm that exclusively works with startups based out of India and the Silicon Valley.',
+            "India's first truly Global Venture Capital Firm. Backing Indian Founders building for Global Markets and disruptive innovation in the US. Web 2 + Web 3",
```

Comment:
Could be because your local `dev` branch isn't updated or an oversight when resolving a merge conflict. If we want to keep the current `dev` version, make sure your local dev branch is up-to-date and run:

```
git checkout analytics
git checkout dev components/about/OurInvestors.vue
git add . && git commit -m 'Copy our investors from dev branch'
```